alert-policies/apache-traffic-server/CPUPercent.yml

```diff
@@ -0,0 +1,40 @@
+# Name of the alert
+name: CPU Percent
+
+# Description and details
+description: |+
+  This alert is triggered if the CPU usage exceeds 95% for 5 minutes.
+# Type of alert
+type: STATIC
+
+# NRQL query
+nrql:
+
+  query: "SELECT average(host.cpuPercent) AS 'CPU used %' FROM Metric"
+
+# Function used to aggregate the NRQL query value(s) for comparison to the terms.threshold (Default: SINGLE_VALUE)
+valueFunction: SINGLE_VALUE
+
+# List of Critical and Warning thresholds for the condition
+terms:
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: ABOVE
+    # Value that triggers a violation
+    threshold: 95
```

Comment:
```suggestion
    threshold: 90
```

Reply:
Hi @pkosanam ,
As per your suggestion, I have made changes. Kindly review it.

Thanks...